### ISSUE TITLE
[BUG] Tru was already initialized. Cannot change database_url

### ISSUE THREAD

**Issue by liuxiaocs7** [truera/trulens#1011]
**Bug Description**
What happened?

```bash
🦑 Tru initialized with db url sqlite:///default.sqlite .

🛑 Secret keys may be written to the database. See the `database_redact_keys` option of Tru` to prevent this.

Tru was already initialized. Cannot change database_url=sqlite:///default.sqlite or database_file=None .

Tru was already initialized. Cannot change database_url=sqlite:///default.sqlite or database_file=None .

Tru was already initialized. Cannot change database_url=sqlite:///default.sqlite or database_file=None .
```

**To Reproduce**
Which steps should someone take to run into the same error? A small, reproducible code example is useful here.

**Expected behavior**

**Relevant Logs/Tracebacks**
Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks. If the issue is related to the TruLens dashboard, please also include a screenshot.

**Environment:**
 - OS: [e.g. MacOS, Windows]
 - Python Version
 - TruLens version
 - Versions of other relevant installed libraries

**Additional context**

**Comment by joshreini1** [comment]
This warning gets printed if you try to repeatedly create the `Tru` object. Can you include all the specifications needed teh first time you create it?

**Comment by joshreini1** [comment]
Closing as this behavior is expected (Tru is a singleton). Please set the url the first time Tru is initialized. If you have additional requirements that make this not possible, please let us know.
